addons/mail/models/mail_thread.py

```diff
@@ -4526,6 +4527,8 @@ def _get_mail_thread_data(self, request_list):
             res['activities'] = self.with_context(active_test=True).activity_ids.activity_format()
         if 'attachments' in request_list:
             res['attachments'] = self._get_mail_thread_data_attachments()._attachment_format()
+            res["areAttachmentsLoaded"] = True
+            res["isLoadingAttachments"] = False
```

Comment:
Don't see where they came from (from the commit this is changed, probably makes sense in global diff)